### PR TITLE
Improve GitHub Discussions rule to be prescriptive about what it can be used for

### PR DESCRIPTION
**Pull request by pierssinclairssw**

### CODE REVIEW

File: rules/use-github-discussions/rule.md [hunk 1]
```diff
@@ -15,33 +15,73 @@ related: []
...
+If you need help with a problem, and Stack Overflow has failed you, then GitHub Discussions offers a way to relay with your team.
+
+## Gauging community sentiment
+Community engagement is hard to achieve but can provide invaluable insights into contentious parts of the application. A post in GitHub Discussions lets you quickly poll all the users of your app.
+
+# Categorisation
+GitHub Discussions allow you to set up categories. Getting these going is important to help users sift through the different topics. Keep the categories simple and easy to follow so that anyone who jumps in will know where to begin.
 
-Now that it's an issue, you can of course associate Pull Requests with that issue and otherwise associate work with that rather than the whole project.
+![Figure: Good Example - TinaCMS is an example of a project with well categories and active GitHub Discussions](TinaCMSDiscussions.png)
```
Comment (SamBWagner) [inline rules/use-github-discussions/rule.md:85]: All the other GitHub screenshots are in dark mode.
1. Please make them consistent

**Review by SamBWagner — CHANGES_REQUESTED** [review]
Checked by @bradystroud 

These changes are a little verbose. The Rule is just trying to convey a useful standard for making content/code/other contributions to a repo/project. I don't think going into detail about what a Discussion is, is that valuable. 

1. Take out all "What" references to Discussion beyond what is baseline for understanding. 
2. Add/morph the other stuff into a section explaining that though this is what we use it for, it can do a whole lot more

**Comment by wicksipedia** [comment]
I feel like the rule should mention the difference between Issues and Discussions

Can we intregrate or quote something from the docs? -  <https://resources.github.com/devops/process/planning/discussions/>

## GitHub Discussions vs. GitHub Issues: When to use each
A common question among developers new to GitHub Discussions goes something like this: When should I use Discussions and when should I use Issues?

**The short answer:** Discussions are great for questions and ideas that require team communication to make a decision, while Issues are defined pieces of work.

**The slightly longer answer:** In any given project or sprint, you have things you need to discuss and those you need to do. Discussions are for—no surprise here—discussing things. Issues are for cataloguing the work you need to do after you’ve reached a decision about how to move forward.

**Comment by SamBWagner** [comment]
@wicksipedia Could we clarify if this is Rule denoting a standard that we're following or an explanation of what Discussions are?

**Review by wicksipedia — APPROVED** [review]
lgtm - just darken the screenshot :)

**Pull request merged**